packages/strapi-plugin-users-permissions/services/Providers.js

```diff
@@ -121,6 +121,25 @@ const getProfile = async (provider, query, callback) => {
     })
     .get();
 
+  const client_id =
+    (strapi.config.provider &&
+      strapi.config.provider[provider] &&
+      strapi.config.provider[provider].client_id) ||
```

Comment:
Can you use `_.get()` here?

Reply:
Thanks @votemike but @petersg83 closed the pull request